schema/v1alpha1/host.proto

```
@@ -0,0 +1,1021 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+syntax = "proto3";
+
+package schema.v1alpha1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "schema/v1alpha1/location.proto";
+import "schema/v1alpha1/state.proto";
+import "schema/v1alpha1/sensor.proto";
```

Buf (1.55.1)

Import "schema/v1alpha1/sensor.proto" is unused.

(IMPORT_USED)